src/snowflake/cli/api/config.py

```diff
@@ -97,8 +98,8 @@ def get_plugins_config() -> dict:
         return {}
 
 
-def is_default_logs_path(path: Path) -> bool:
-    return path.resolve() == Path(str(_DEFAULT_LOGS_CONFIG["path"])).resolve()
+def is_default_logs_path(path: SecurePath) -> bool:
+    return path.path.resolve() == Path(str(_DEFAULT_LOGS_CONFIG["path"])).resolve()
```

Comment:
Hm, this is a bit weird, isn't it? Can we do `resolve` on `path`? Can `SecurePath` behave like a normal `Path`?